src/coreclr/vm/frames.cpp

```diff
@@ -464,6 +464,21 @@ void Frame::PopIfChained()
 }
 #endif // TARGET_UNIX && !DACCESS_COMPILE
 
+#if !defined(TARGET_X86) || defined(TARGET_UNIX)
+void Frame::UpdateFloatingPointRegisters(const PREGDISPLAY pRD, TADDR targetSP)
```

Comment:
`targetSP` argument is unused.

Reply:
`this`  is not used either. It can be a static method.

Reply:
Fixed both